samples/bluetooth/direct_test_mode/src/dtm.c

```diff
@@ -106,11 +130,35 @@ BUILD_ASSERT(NRFX_TIMER_CONFIG_LABEL(ANOMALY_172_TIMER_INSTANCE) == 1,
 #define PACKET_STATIC_LEN         0
 /* Base address length in bytes. */
 #define PACKET_BA_LEN             3
-
+/* CTE IQ sample data size. */
+#define DTM_CTE_SAMPLE_DATA_SIZE  128
 /* Vendor specific packet type for internal use. */
 #define DTM_PKT_TYPE_VENDORSPECIFIC  0xFE
 /* 1111111 bit pattern packet type for internal use. */
 #define DTM_PKT_TYPE_0xFF            0xFF
+/* Response event data shift. */
+#define DTM_RESPONSE_EVENT_SHIFT 0x01
+/* Maximum number of payload octets that the local Controller supports for
+ * transmission of a single Link Layer Data Physical Channel PDU.
+ */
+#define NRF_MAX_PAYLOAD_OCTETS   0x00FF
+
+/* Maximum transmit or receive time, in microseconds, that the local
+ * Controller supports for transmission of a single
+ * Link Layer Data Physical Channel PDU, divided by 2.
+ */
+#if defined(NRF52840_XXAA) || defined(NRF52833_XXAA) || defined(NRF52811_XXAA) || defined(NRF52820_XXAA)
+#define NRF_MAX_RX_TX_TIME 0x2148
+#else
+#define NRF_MAX_RX_TX_TIME 0x424
+#endif /* defined(NRF52840_XXAA) || defined(NRF52833_XXAA) || defined(NRF52811_XXAA) || defined(NRF52820_XXAA) */
```

Comment:
All configurations that depend on different SoCs, can be kept in the dedicated header file

Reply:
Done